docs/tutorials/ifci_px4_flight_controller.rst

```diff
@@ -0,0 +1,304 @@
+        :align: center
+        :scale: 50
+        :alt: Trimmed Code
+
+        Trimmed Code
+
+Flashing PX4 to Your Flight Controller
+======================================
+
+Once built, you must program your flight controller with your compiled ``.px4`` file. To do this, open QGroundControl, go to the vehicle settings menu and enter the 'Firmware' menu. Once there, plug in your board, select the 'Advanced Settings' checkbox, and then the 'Custom Firmware' option.
+
+.. figure:: ../_static/tutorial_images/ifci_px4_flight_controller/custom_firmware.png
+    :align: center
+    :scale: 50
+    :alt: Custom Firmware Selection
+
+    Custom Firmware Selection
+
+Once you press 'OK', a file explorer should appear. Find the ``your-flight-control-board_default.px4`` file that you built and select it. The flashing process should begin. 
+
+Enabling IFCI on Your Flight Controller
+===================================================
+
+Once the flashing is complete, connect to your flight controller with QGroundControl and go to the parameters menu. In the parameters menu :red:`search for 'vertiq'`. The parameter ``VERTIQ_IO_CFG`` :blue:`should appear`. Once you find this, :green:`set it to the serial port that you plan on using`, save, and reboot the flight controller as instructed by QGroundControl.
+
+.. figure:: ../_static/tutorial_images/ifci_px4_flight_controller/vertiq_io_enable.png
+    :align: center
+    :scale: 50
+    :alt: Enabling Vertiq IO
+
+    Enabling Vertiq IO
+
+After reboot, and with Vertiq IO enabled, you should now see a :blue:`Vertiq IO` submenu in the QGroundControl parameter settings. Adjust the :red:`VERTIQ_BAUD` parameter to match what your modules will be using. In this tutorial we will be using a baud rate of 921600 which is what we recommend.
+
+.. figure:: ../_static/tutorial_images/ifci_px4_flight_controller/vertiq_io_settings.png
+    :align: center
+    :scale: 50
+    :alt: Vertiq IO Submenu
+
+    Vertiq IO Submenu
+
+Now your Vertiq modules must be configured for proper communication with the flight controller.
+
+Configuring Your Vertiq Modules for Use with IFCI and PX4
+========================================================
+To use your Vertiq modules properly with IFCI, your modules must be flashed with a compatible firmware version. Please consult your module's family page to find if your module supports IFCI. Once flashed with appropriate firmware, connect each module **individually** to IQControlCenter and set the :blue:`UART Baud Rate` and the :red:`Module ID`. As stated previously, we recommend that you use a baud rate of 921600. Both of these parameters will cause the motor to reboot when set, so make sure you reconnect to the motor after each one is set. Ensure that each module connected to the flight controller is set to a different, unique module ID. For this tutorial we will be using the module IDs 0, 1, 2, and 3.
```

Comment:
does  changing the baud rate cause the motor to reboot?

Reply:
You're right, it just needs to reconnect. I've reworded it and added a thing saying to do module ID first to avoid needing to change the control center baud rate.